packages/platform/testing/html-parser.js

```diff
@@ -0,0 +1,21 @@
+import { prettyPrint } from 'html';
+import htmlParser from 'node-html-parser';
+
+/**
+ * Parses HTML and returns a root element.
+ *
+ * @param {string} html
+ * @param {Partial<import('node-html-parser').Options> & { rootElement?: string }} [options={}]
+ * @returns {HTMLElement}
+ */
+export const parseHtml = (html, { rootElement = 'main', ...options } = {}) => {
+	const prettifiedHtml = prettyPrint(
```

Comment:
Is `prettyPrint` required for nice diffs?

Reply:
nicer snapshots... nunjucks can end up inserting a lot of whitespace because of the way it interprets it's formatting, so I'm using the parser to strip the whitespace entirely, and then pretty print to format it nicely for snapshotting

Reply:
I thought so.

Reply:
I would say it's worth it even if it's slower, because the primary reason to do it is not so it looks nicer in the file but so that snapshots can be read by a developer much easier when asserting them or doing diffs...